torchci/lib/fetchFailureSamples.ts

```diff
@@ -0,0 +1,23 @@
+import getRocksetClient from "lib/rockset";
+import rocksetVersions from "rockset/prodVersions.json";
+
+import { JobData } from "./types"
+
+export default async function fetchFailureSamples(capture: string | string[]): Promise<JobData[]> {
```

Comment:
I feel like we should make it either a string or string array, not both.

Reply:
I'm gonna leave it cuz this was from the code before--maybe can fix in a different PR